### PR TITLE
Make upstream messages work on ios

### PR DESCRIPTION
**Pull request by tolu360**
In conforming with the method signatures as is the case with the
android version, I limited the arguments for `send()` to just
`senderId` and `payload`. This can still be improved upon by adding
`sendDataMessageSuccess` and `sendDataMessageFailure` to events being
fired across the bridge to Js.

### CODE REVIEW

File: ios/RNFIRMesssaging.m [hunk 1]
```diff
@@ -99,6 +99,14 @@ - (void)setBridge:(RCTBridge *)bridge
   [[NSNotificationCenter defaultCenter]
    addObserver:self selector:@selector(onTokenRefresh)
    name:kFIRInstanceIDTokenRefreshNotification object:nil];
+
+  [[NSNotificationCenter defaultCenter]
```
Comment (evollu) [inline ios/RNFIRMesssaging.m]: are these needed? I don't find the documentation

Comment (tolu360) [inline ios/RNFIRMesssaging.m]: Hi @evollu,

They are not `mandatory`, but would be `nice to have`. They are just event listeners to know if the message was successfully delivered on the app server or not. They can also be useful when debugging with XCode console. You can find further documentation at https://developers.google.com/cloud-messaging/ios/ios-migrate-fcm and https://firebase.google.com/docs/cloud-messaging/ios/upstream.

Cheers

**Comment by evollu** [comment]
Thanks, I see the docs.

**Comment by evollu** [comment]
@tolu360 do you mind update readme and add some example code for it?

**Comment by tolu360** [comment]
Sure thing, I would add usage example to the readme shortly.

**Pull request merged**